software/utils/debug/debug_cli.py

```diff
@@ -553,10 +554,10 @@ def do_get(self, line):
                 else:
                     print(dark_orange(text))
             return
-
-        variable_md = self.interface.variable_metadata[cl[0]]
-        val = self.interface.variable_get(cl[0], raw=raw, fmt=fmt)
-        print(variable_md.print_value(val))
+        else:
+            variable_md = self.interface.variable_metadata[cl[0]]
+            val = self.interface.variable_get(cl[0], raw=raw, fmt=fmt)
```

Comment:
Shouldn't we reuse `var_name` here as well?

Reply:
Good catch. Fixed.